src/components/AsideHeader/AsideHeader.module.scss

```
@@ -218,6 +249,57 @@ $block: '.#{variables.$ns}aside-header';
+        &_unified {
+            flex: none;
+        }
+    }
+
+    &__quick-access-title {
+        @include mixins.text-caption-2;
+
+        color: var(--g-color-text-secondary);
+        margin-inline: var(--_--item-margin-inline);
+        padding-inline: calc(
+                var(--_--gn-aside-header-density-item-height, #{variables.$item-height}) +
```

Stylelint (17.14.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)